smrf/tests/data/test_modifiers.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+
+import numpy as np
+import pandas as pd
+
+from smrf.data import modifiers
+from parameterized import parameterized
+
+
+input_array = np.array([
+    [1.0, 1.1, 1.2],
+    [3.1, 3.2, 5.7],
+    [0.1, -1.1, 2.0]
+])
+
+input_series = pd.Series([10.2, 42.0, 5.1])
+
+
+class TestModifiers(unittest.TestCase):
+    @parameterized.expand(
+        [
+            ('input', {}, None),
+            ('input', {'input_scalar_type': 'factor'}, 'input_scalar_factor'),
+            ('output', {'output_scalar_type': 'factor'},
+             'output_scalar_factor'),
+            ('output', {}, None)
+        ]
```

Comment:
How about a scenario with unknown keys to test error cases?

Reply:
Good call, will do!